xbmc/SortFileItem.h

```diff
@@ -40,6 +40,8 @@ typedef enum {
   SORT_METHOD_YEAR,
   SORT_METHOD_VIDEO_RATING,
   SORT_METHOD_DATEADDED,
+  SORT_METHOD_ALBUM_DATEADDED,
+  SORT_METHOD_ARTIST_DATEADDED,
```

Comment:
this will break the int we use in plugins http://mirrors.xbmc.org/docs/python-docs/14.x-helix/xbmcplugin.html#-addSortMethod